index.js

```diff
@@ -7,20 +7,18 @@ const getNextOptions = (options, mentions) =>
     ? options
     : merge(options, { max_id: pipe(last, prop('id_str'), dec)(mentions) });
 
-function accumulate(get, options, mentions, cb) {
+function accumulate(client, options, mentions) {
   const nextOptions = getNextOptions(options, mentions);
-  get(nextOptions, (err, res) => {
-    if (err) return cb(err);
+  return client.get('statuses/mentions_timeline', nextOptions).then(res => {
     const accumulatedMentions = concat(mentions, res);
     return (isEmpty(res))
-        ? cb(null, accumulatedMentions)
-        : accumulate(get, nextOptions, accumulatedMentions, cb);
+        ? accumulatedMentions
```

Comment:
this is not breaking chain. If it returns a value (not promise), then `accumulatedMentions` is passed along the chain to the next `then`